justfile

```diff
@@ -4,6 +4,18 @@ test:
   source .env
   set +o allexport
   go test -count=1 -v ./pinecone
+test-integration:
+    #!/usr/bin/env bash
+    set -o allexport
+    source .env
+    set +o allexport
+    go test -count=1 -v -run TestIntegrationClient ./pinecone
```

Comment:
This runs all tests that are methods on the `ClientTestsIntegration` object.

Reply:
The idea here is that we can have separate `structs` for integration tests vs. unit tests. Right now, we only have one for integration tests, but we could have one for unit tests in the future. If we do that, we would change the `test-unit` command in the `justfile` to call whatever function we create that actually implements `suite.Run(....)`.

Reply:
I know you have another PR for adding tests to index connection as well - is there a way to just use "Integration" for this like the `test-unit` command? It feels like this will only run integration tests covered by `TestIntegrationClient` rather than any that include `Integration`.

Reply:
You're right -- for some reason in my head there was a reason I did it differently for integration vs unit tests, but now I can't remember... and it doesn't seem to matter! I'll change :)